01week/rockPaperScissors.js

```diff
@@ -10,10 +10,34 @@ const rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
-
+  if (hand1 === hand2){
+    return "It's a tie!";
+  }else if (hand1 === 'rock' && hand2 === 'paper'){
```

Comment:
This logic is sound, however, you seem to be repeating yourself. Refactor these if statements so that you are not repeating yourself at all.